doc/architecture/database_schemata.md

```diff
@@ -87,8 +87,62 @@ species }|--|| genus : ""
 genus }|--|| subfamily : ""
 subfamily }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_variant "NOT NULL"
+  VARCHAR version_name
+  BOOLEAN is_inactive
+  DATE last_visit
+  INT honors
+  INT visits
+  INT harvested
+  DATE creation_date
+  DATE deletion_date
+  INT zoom_factor
+  BYTEA background
+  INT background_scale
+  GEOGRAPHY geo_data
+  INT epsg_code
+}
+
+users {
+  INT id PK
+  VARCHAR nc_uid
+  DATE free_until
+  VARCHAR app_language
+  DATE member_since
+  VARCHAR experience
+  VARCHAR preferences
+  GEOGRAPHY location
+  INT membership_points
+}
+
+blossoms {
+  INT id PK
+  VARCHAR title
+  VARCHAR description
+  VARCHAR condition
+  TRACKS track
+  BYTEA icon
+  BOOLEAN is_seasonal
+}
 
+enum_tracks {
+  VARCHAR Beginners_Track
+  VARCHAR Seasonal_Track
+  VARCHAR Completionist_Track
+  VARCHAR Expert_Track
+}
+
+blossoms_gained {
+  INT id PK
+  INT times_gained
```

Comment:
What would be also interesting is to know which tracks are too easy/too difficult, so statistics how many users are how far ahead in which track. Is this derivable from this data?

Reply:
It should be possible to create a detailed report from the blossoms_gained data via Joins with the blossoms and the users table. SQL alone might already work, even though querying the data and then processing them externally would probably be simpler.

I will also include a timestamp attribute in the table, so that we can also add a time constraint to this report.

Reply:
You are right. 'gained_date' should probably be an array of dates, so we can keep track of every time it was earned.